src/Arriba/Arriba/Diagnostics/Tracing/ArribaLogs.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+
+namespace Arriba.Diagnostics.Tracing
+{
+    public class ArribaLogs
```

Comment:
Should probably be "ArribaLog" - singular

Reply:
Done.